examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -2466,6 +2503,124 @@ impl FrontendNode {
         expect_matches!(result, Err(_));
         Ok(())
     }
+
+    fn test_grpc_server_pseudo_node_privilege(&mut self) -> TestResult {
+        // gRPC server pseudo node can be created even with Top confidentiality.
+        let top_label = oak_abi::label::confidentiality_label(oak_abi::label::top());
+        let config = oak::node_config::grpc_server(ADDITIONAL_TEST_SERVER_ADDR);
+        let (wh, rh) = oak::channel_create("Test", &top_label).expect("could not create channel");
+        expect_eq!(
+            Ok(()),
+            oak::node_create("grpc_server", &config, &top_label, rh)
+        );
+        expect_eq!(Ok(()), oak::channel_close(rh.handle));
+        expect_eq!(Ok(()), oak::channel_close(wh.handle));
+        Ok(())
+    }
+
+    fn test_http_server_pseudo_node_privilege(&mut self) -> TestResult {
+        // HTTP server pseudo node can be created even with Top confidentiality.
+        let top_label = oak_abi::label::confidentiality_label(oak_abi::label::top());
+        let config = oak::node_config::http_server(ADDITIONAL_TEST_SERVER_ADDR);
+        let (wh, rh) = oak::channel_create("Test", &top_label).expect("could not create channel");
+        expect_eq!(
+            Ok(()),
+            oak::node_create("http_server", &config, &top_label, rh)
+        );
+        expect_eq!(Ok(()), oak::channel_close(rh.handle));
+        expect_eq!(Ok(()), oak::channel_close(wh.handle));
+        Ok(())
+    }
+
+    fn test_grpc_client_pseudo_node_privilege(&mut self) -> TestResult {
+        // gRPC client pseudo node can be created with TLS endpoint confidentiality tag matching its
+        {
+            // URI authority.
+            let label = oak_abi::label::confidentiality_label(oak_abi::label::tls_endpoint_tag(
+                "localhost:7878",
```

Comment:
Perhaps this, or the port number should be changed into a const.

Reply:
Made the address a const and now extract the authority by parsing it as a URI.